src/main/java/utility/Ui.java

```diff
@@ -0,0 +1,51 @@
+package utility;
+
+/**
+ * UI class required for displaying information.
+ */
+public class Ui {
+
+    /**
+     * Prints out the welcome text.
+     */
+    public void showWelcome() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        String msg = "____________________________________________________________\n"
+                + "Hello I'm Duke\n"
+                + "Your personal smart assistant\n"
+                + "____________________________________________________________";
```

Comment:
Perhaps these strings could be abstracted as constant variables at the top of the class? Would be easier to edit and cleaner too!